website/docs/workloads/mlperf/mlperf-profiles.md

```diff
@@ -38,6 +38,7 @@ Runs the MLPerf benchmark workload to test GPU performance.
   | LinuxDriverVersion     | Optional. Version of GPU driver that needs to be installed. | 510  |
   | LinuxLocalRunFile     | Optional. Link to download specified CUDA and GPU driver versions. | https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run  |
   | RequireCustomSystemSupport | Optional. This enables additional A100_PCIe_80GBx4 system support that was not supported by github repo of MLPerf. Ones that are supported by github repo of MLPerf are still supported. | true |
```

Comment:
I would like to see if this could be removed now. Is MLPerf supporting A100x4 now?

Reply:
No, they are not.